scripts-2.0/components/PopupMenu/WorkingTimerSection/index.tsx

```diff
@@ -20,7 +20,7 @@ const WorkingTimerSection: React.FC<WorkingTimerSectionInterface> = (props) => {
     const [isTimerWorking, setIsTimerWorking] = useState(false);
     const [entry, setEntry] = useState<Entry>(props.entry);
     const [currentTimer, setCurrentTimer] = useState(timeFormatter.formatToDuration(0));
-    const [durationFormat, setDurationFormat] = useState(DURATION_FORMATS.CLASSIC);
+    const [durationFormat, setDurationFormat] = useState(DURATION_FORMATS.CLASSIC_WITH_SECONDS);
```

Comment:
format should be changed to `DURATION_FORMATS.HHMMSS`
use `DEFAULT_FORMAT` from `scripts-2.0/TimeFormatter.js` (change it first :) )